library/ziti.c

```diff
@@ -96,7 +96,10 @@ static size_t parse_ref(const char *val, const char **res) {
     if (val != NULL) {
         if (strncmp("file:", val, 5) == 0) {
             // load file
-            *res = val + strlen("file://");
+            struct tlsuv_url_s url;
+            tlsuv_parse_url(&url, val);
+            size_t start = strlen(val) - strlen(url.path);
+            *res = val + start;
```

Comment:
isn't this just `*res = url.path`?

Reply:
wouldn't that pointer (url.path) go out of scope as soon as the function returns? it would need to use val, no?

Reply:
actually yeah i see what you mean now i thought about it a bit more.